userspace/libscap/scap-int.h

```diff
@@ -347,6 +347,12 @@ const char *scap_strerror(scap_t *handle, int errnum);
 #define MAX(X,Y) ((X) > (Y)? (X):(Y))
 #endif
 
+#undef  TRUE
+#define TRUE 1
+
+#undef  FALSE
+#define FALSE 0
+
```

Comment:
We use stdbool, so you should remove this code and use lower-case `true` and `false` instead.

Reply:
I will change TRUE and FALSE to true and false.